packages/legend-query/src/components/QueryBuilderPropertyExpressionEditor.tsx

```diff
@@ -84,6 +86,14 @@ const DerivedPropertyParameterValueEditor = observer(
       [handleDrop],
     );
     const resetParameterValue = (): void => {
+      if (
```

Comment:
Looking back, this seems wrong to me, this is the reset button, so it should only reset 1 parameter value rather than resetting all of them as you call `fillDerivedPropertyParameterValues`. I think what we should do instead is to use the similar logic we have in `fillDerivedPropertyParameterValues` `forEach` block